calicoctl/resourcemgr/networkpolicy.go

```diff
@@ -29,7 +29,7 @@ func init() {
 		true,
 		[]string{"networkpolicy", "networkpolicies", "policy", "np", "policies", "pol", "pols"},
 		[]string{"NAME"},
-		[]string{"NAME", "ORDER", "SELECTOR", "NAMESPACE"},
+		[]string{"NAME", "ORDER", "SELECTOR"},
 		map[string]string{
 			"NAME":      "{{.ObjectMeta.Name}}",
 			"NAMESPACE": "{{.ObjectMeta.Namespace}}",
```

Comment:
can you add a quick comment here saying NAMESPACE is implicitly added to the template in  GetTableTemplate so humans don't delete this in future thinking it's not in the headings so we don't need it

Reply:
Another option I considered was to have namespace in the ps and wide and then in the GetTableTemplate remove the field from the headings if printNamespace was false. Would that be preferable?

Reply:
I think this is fine, not too fussed either way tho.